Automattic/jetpack
# Update/blogroll item use url as name substitute

## Proposed changes:

Updates the blogroll item recommendations to use the URL as the name if the name attribute does not exist.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Checkout branch
* Build jetpack plugin and sync to sandbox
* Navigate to the editor and add a new blogroll (ensure the URL appears the name if the name attribute doesnt exist for a specific site recommendation).
* Test on Atomic using Jetpack beta plugin
